code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -42,3 +42,4 @@
 	var/list/stasis_sources = list()
 	var/stasis_value
 	var/pain_immune = FALSE //for special cases where something permanently removes a mob's ability to feel pain
+	var/accumulated_pain = 0 //for healing pain at the end of a panotoxin poisoning
```

Comment:
Adding a variable dedicated to one specific drug/effect to track how much it has done isn't ideal, you can however generalize this by making it an associative list, with each drug singleton as the key and the damage inflicted as the value, it also needs to have a proper DMdoc

Reply:
I thought I might be doing something wrong. I don't understand what you're saying, though. I don't know what an associative list means, or what keys and values mean in relation to associative lists, or what DMdocs are.

Reply:
Did my own research. I don't know if what I did actually added a DMdoc, but it uses a list now.

Reply:
DMdoc is as follows, for a variable:

```
/// This is a dmdoc
var/variable
```

```
/**
 * This is a
 * multiline dmdoc
 * for a variable
 *
 * With spaced lines
 */
var/variable
```